Importing `imports/server/XXX.js` from `server/main.js` causes an ESLint error on Windows

I assume that Windows-specific directory delimiter `\` causes this problem.
I created a reproduction repository. I got the following errors.
https://github.com/exKAZUu/test-eslint-import-resolver-meteor

```
C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor>npm run eslint

> test-eslint-import-resolver-meteor@ eslint C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor
> eslint . --fix

The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

C:\Users\exKAZUu\Desktop\test-eslint-import-resolver-meteor\server\main.js
  3:32  error  Unable to resolve path to module '../imports/server/value34'  import/no-unresolved

✖ 1 problem (1 error, 0 warnings)
```

--- a/index.js
+++ b/index.js
@@ -54,16 +54,16 @@ function packageFilter(pkg, path, relativePath) {
 function findMeteorRoot(start) {
   start = start || module.parent.filename
   if (typeof start === 'string') {
-    if (start[start.length-1] !== path.sep) {
-      start+=path.sep
+    if (start[start.length-1] !== '/') {
+      start += '/'
     }
-    start = start.split(path.sep)
+    start = start.split('/')
   }
   if(!start.length) {
     throw new Error('.meteor not found in path')
   }
   start.pop()
-  var dir = start.join(path.sep)
+  var dir = start.join('/')
   try {
     fs.statSync(path.join(dir, '.meteor'))
     return dir

